### ISSUE TITLE
'=' is an unexpected token

### ISSUE THREAD

**Issue by kwild1140** [Microsoft/mattercenter#728]
I have been trying to complete the deploy.ps1 script, but keep getting the following message: '=' is an unexpected token. The expected token is ';'. Line 1, position 107. The problem is, this message is showing up randomly as the script runs. For instance, one time I will get the error while 'creating taxonomy hierarchy', while other times 'creating taxonomy hierarchy' completes successfully and I'll get the message during the 'create columns and content types' step. The next time I run the script both of those will complete successfully and the error message will be generated in some other step. Bottom line, the script is not completing successfully, but I cant isolate the root cause because it doesn't occur at the same time every time. I am deploying matter center to a 'demos.microsoft.com' tenant as practice for an eventual production deployment. Any ideas?

**Comment by amitabhm21** [comment]
Can you please make sure that account that you are using to run the script has Administrative privileges on this Tenant ?

Also make sure that you get the latest code from the Repo.

**Comment by kwild1140** [comment]
Hi amitabhm21... got the latest version of the code and made sure the account has admin privileges... got the same results. I am using one of my Microsoft Demos sites to attempt this installation so I'm sure the 'admin' account has the necessary credentials. 

What's next? 

Thanks.

**Comment by kwild1140** [comment]
Hi amitabhm21... can you think of any other steps I can take to resolve this issue? I started a fresh build and deploy today and ended up with the same error message. However, now it happens all the time on the first step in the process... creating the groups.

**Comment by premchandpl** [comment]
@kwild1140 Can we have skype meeting and we can look into the issue. By the way I am matter center developer and my skype id is "premchand_76@hotmail.com"

**Comment by kwild1140** [comment]
Hi @premchandpl ... thanks for the offer. For whom are you a matter center developer?

**Comment by premchandpl** [comment]
@kwild1140 I am part of the developer team who developed this matter center application

**Comment by kwild1140** [comment]
Thanks @premchandpl ... I need to be on a call from 3-4 CST. When would you be available to chat?

**Comment by premchandpl** [comment]
@kwild1140 I am in PST zone. May be we will chat on Friday 11-12 PST

**Comment by kwild1140** [comment]
I'll set up a skype call shortly... thanks.

**Comment by kwild1140** [comment]
@premchandpl I sent a skype invite... I guess you didn't get it. My afternoon is packed the rest of the day. I'll reach out next week and see what I can do to get us together to discuss. Thanks.

**Comment by amitabhm21** [comment]
Is this issue still open? @kwild1140

**Comment by kwild1140** [comment]
No... did not get a resolution either. We're thinking it has something to do with the way Azure is synched / linked to the demos.microsoft.com demo sites. We gave up.
